parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -208,6 +275,8 @@ c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-containe
 `$global:CNIConfig = "$global:CNIConfig"
 `$global:HNSModule = "$global:HNSModule"
 `$global:VolumePluginDir = "$global:VolumePluginDir"
+`$global:MaxPods="$global:MaxPods"

```

Comment:
Where is MaxPods used?

Reply:
Not yet, but could be used in the future. Should I remove it for now?

Reply:
Yes, please remove it.